transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/accesslog/AccessLogConfiguration.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.transport.rest.vertx.accesslog;
+
+import com.netflix.config.DynamicPropertyFactory;
+
+public final class AccessLogConfiguration {
+  private static final String BASE = "servicecomb.accesslog.";
+
+  private static final String ACCESSLOG_ENABLED = BASE + "enabled";
+
+  private static final String ACCESSLOG_PATTERN = BASE + "pattern";
+
+  public static final AccessLogConfiguration INSTANCE = new AccessLogConfiguration();
+
+  private AccessLogConfiguration() {
+
+  }
+
+  public boolean getAccessLogEnabled() {
+    String enabled = getProperty("false", ACCESSLOG_ENABLED);
+    return Boolean.parseBoolean(enabled);
```

Comment:
while not just DynamicPropertyFactory.getInstance().getBooleanProperty?

Reply:
This has been changed to use DynamicPropertyFactory.getInstance().getBooleanProperty()